src/components/Question.js

```diff
@@ -23,42 +45,49 @@ const Question = (props) => {
 				name={props.questionName}
```

Comment:
Can you also look into increasing the size the Question text in the Form Label component?
At Line 42.